openstack/clustering/v1/webhooks/doc.go

```diff
@@ -0,0 +1,11 @@
+/*
+Package webhooks enables triggers an action represented by a webhook from the OpenStack
+Clustering Service.
+
+Example to Trigger webhook action
+	result := webhooks.Trigger(webhookClient, "my-webhook")
+	if result.Err != nil {
+		panic(result.Err)
+	}
```

Comment:
Since you're defining `TriggerOpts` in the code, wouldn't the above example look more like:

```go
result := webhooks.Trigger(clusterClient, "webhook-uuid", nil)
```

Additionally, `result.Err` is correct, but I think it would be preferable to use the more common form:

```go
result, err := webhooks.Trigger(clusterClient, "webhook-uuid", nil).Extract()
if err != nil {
  panic(err)
}
```

Reply:
Agree, changed to the second form